test/test_util.py

```diff
@@ -1,18 +1,15 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-import io
-import locale
+import locale # noqa F401 imported but unused see #1106
```

Comment:
The errors in #1106 seem unrelated to this. Can you try to remove this again, and see if the error is relevant this time please.

Reply:
I removed the `import locale` and there are errors for the [CI / test (ubuntu-latest, pypy-3.7) (pull_request) ](https://github.com/airspeed-velocity/asv/runs/5489967552?check_suite_focus=true) and the [CI / test (ubuntu-latest, pypy-3.8) (pull_request)](https://github.com/airspeed-velocity/asv/runs/5489967597?check_suite_focus=true). The tests are passed for the `test/test_subprocess.py` and `test/test_util.py` files in both cases.

<img width="1082" alt="image" src="https://user-images.githubusercontent.com/13194519/157584274-65ea991c-e0ab-4d5a-97c2-769a40c34f7f.png">

<img width="1080" alt="image" src="https://user-images.githubusercontent.com/13194519/157584419-0edc7f36-23ac-452d-a2e4-af08fef35553.png">

Reply:
These failures seem unrelated. Can you just remove the `locale` import from this PR.

Reply:
Done!